pkg/services/action.go

```diff
@@ -99,28 +99,24 @@ func NewSubmitSystemIntake(
 		updatedTime := config.clock.Now()
 		intake.UpdatedAt = &updatedTime
 		intake.Status = models.SystemIntakeStatusINTAKESUBMITTED
-
-		if intake.AlfabetID.Valid {
-			err := &apperrors.ResourceConflictError{
-				Err:        errors.New("intake has already been submitted to CEDAR"),
-				ResourceID: intake.ID.String(),
-				Resource:   intake,
+		intake.RequestFormState = models.SIRFSSubmitted
+
+		intakeSubmittedToCedar := intake.AlfabetID.Valid // When submitted to CEDAR, the AlfabetID gets set. If set, we currently don't re-submit it
+
+		if !intakeSubmittedToCedar {
```

Comment:
I really like this change in approach to just skip the submission to CEDAR rather than returning an error, it's a much more graceful way to handle things.